### DISCUSSION TITLE
LogTo in .net microservice and micro UI

### DISCUSSION THREAD

**Discussion by vitalybrandes** [logto-io/logto#5893] [General]
Hi all, I have a question related to using LogTo in .net core microservice architecture  where I am also using microUI as containers. I am lost in using the API as a resource to get access token.  I need access token to be available for me in UI. Using access token I call multiple apis for the same ui. Does the access token for user have access to multiple resources and can I forward it to apis? If yes, is there any demo of code? For now I found that in razor pages example the access token only available in UI (page itself) what if I have DO service that need this token to contact api?

**Comment by xiaoyijun** [comment]
Hi @vitalybrandes , could you please provide more context about your software architecture? I'm wondering what does the UI means.

**Reply by vitalybrandes** [reply]
@xiaoyijun  Sure. I have large microservice architecture that include mostly .net fremework ,.net web api, razor pages and blazor wasm.
Regarding the microfrontedUI , my user login to some service X, and using token send api request to different microservices, where the "Resource" is different. I trying to make this call in the .net razor pages and didn't find how to get multiple resources using. LogTo .net sdk.
Also, while user want to move from one ui to different ui, let's say from microUI product to microUI orders, the session should support it with no need to authenticate again, if user allowed, as this is the same domain.
Maybe all this possible I am just confused hot to.make it work 🤷🏼‍♂️

**Comment by xiaoyijun** [comment]
Hi @gao-sun  , could you please take a look about this issue, since I have no context about the .net tech stack and also the .net SDK.
